### PR TITLE
Stack marker file results rather than outputing to a directory

### PR DESCRIPTION
**Pull request by pinin4fjords**
As requested by @nh3 , this PR stacks k-wise outputs from the marker detection routines of SC3 such that a single file is output.

### CODE REVIEW

File: sc3-sc3-calc-biology.R [hunk 1]
```diff
@@ -45,14 +45,13 @@ option_list = list(
   )
 )
 
-opt <- wsc_parse_args(option_list, mandatory = c('input_object_file', 'output_object_file', 'output_dir'))
+opt <- wsc_parse_args(option_list, mandatory = c('input_object_file', 'output_object_file', 'output_text_file'))
```
Comment (nh3) [inline sc3-sc3-calc-biology.R]: LGTM! One small ask, could you remove `output_text_file` from list of mandatory options, as the information is already in the object and the text file can be large given the length of Ks.

Comment (pinin4fjords) [reply sc3-sc3-calc-biology.R]: Okay, done- look good?

**Review by nh3 — APPROVED** [review]
Thanks @pinin4fjords!

**Comment by pinin4fjords** [comment]
No worries @nh3 - do you need this ASAP? Do we need to release?

**Comment by nh3** [comment]
That would be ideal, as the sc3 workflow needs this fix to get to release. Thank you.

**Pull request merged**